docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py

```diff
@@ -0,0 +1,46 @@
+"""
+This is an example script for how to create a custom Action.
+
+To test, run:
+pytest --docs-tests -k "docs_example_create_a_custom_action" tests/integration/test_script_runner.py
+"""
+
+# EXAMPLE SCRIPT STARTS HERE:
+
+# <snippet name="docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py - full code example">
+
+from typing import Literal
+
+from typing_extensions import override
```

Comment:
**Note for technical reviewer:** I got an error about override not being defined so I added this based on other code I see in the repo. It resolved the error so I think it's right but let me know if there's anything amiss here.